cupy/core/_fusion_analysis.py

```diff
@@ -0,0 +1,612 @@
+import six
+
+import numpy
+
+from cupy.core import _errors
+from cupy.core import _kernel
+from cupy.core import core
+from cupy.core import _fusion_interface
+from cupy.core import _fusion_variable
+from cupy.core._fusion_variable import _FusionCudaScalar
+from cupy.core._fusion_variable import _FusionCudaArray
+from cupy.core._fusion_variable import _FusionVariableSet
+from cupy.core import _fusion_shape
+from cupy.core import _fusion_device_func
+from cupy.core import _fusion_op
+from cupy.core._fusion_emit_code import _CodeBlock
+from cupy.core import _fusion_runtime
+from cupy.core import _fusion_optimization as _opt
```

Comment:
I think you don't have to use a short name `_opt` because it's used only a few times.